test/DynamicExpresso.UnitTest/GithubIssues.cs

```diff
@@ -510,7 +510,7 @@ public void GitHub_Issue_221_Case_insensitivity()
 
 			// case insensivity outside lambda expressions
 			Assert.IsFalse(interpreter.Eval<bool>("dateinthepast > now()")); // identifier
-			Assert.IsTrue(interpreter.Eval<bool>("dateinthepast is datetimeoffset)")); // known type
+			Assert.IsTrue(interpreter.Eval<bool>("dateinthepast is datetimeoffset")); // known type
```

Comment:
One the test had an invalid parenthesis to workaround the parser's bug 😅

Reply:
Didn't remember 😅